src/compile/format.ts

```diff
@@ -30,8 +31,6 @@ function customFormatExpr(formatType: string, field: string, format: Format) {
   return `${formatType}(${field}${format ? `, ${stringify(format)}` : ''})`;
 }
 
-export const BIN_RANGE_DELIMITER = ' \u2013 ';
```

Comment:
Nice to not have to export something that isn't reused on the outside 👍

Reply:
We had this twice.